playbarComponent/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+  entry: path.resolve(__dirname, 'client', 'src', 'index.jsx'),
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'public')
+  },
+  resolve: {
+    extensions: ['.js', '.jsx']
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /(node_modules)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react']
+          }
+        }
+      }
+    ]
+  },
```

Comment:
you'll need to configure your webpack to take in css files if you plan on using css modules.